pl-faded-parsons.js

```diff
@@ -921,6 +875,17 @@ class ParsonsWidget {
     prev_log.push(entry);
     s.val(JSON.stringify(prev_log));
   }
+  generateMockPLData() {
+    const txInputs = [
+      $(this.config.storage),
+      $(this.config.logStorage),
+    ];
+    const data = {};
+    for (let inp of txInputs) {
+      data[inp.attr('name')] = inp.val();
+    }
+    return JSON.stringify({ "raw_submitted_answers": data });
+  }
```

Comment:
Note: add description that says that this is a _debugging_ tool, not a part of the primary function